photon-client/src/views/PipelineViews/InputTab.vue

```diff
@@ -22,10 +22,27 @@
       @input="handlePipelineData('cameraBrightness')"
       @rollback="e => rollback('cameraBrightness', e)"
     />
+    <CVswitch
+      v-model="cameraAutoExposure"
+      class="pt-2"
+      name="Auto exposure"
+      @input="handlePipelineData('cameraAutoExposure')"
+    />
+    <CVslider
+      v-if="cameraGain >= 0"
+      v-model="cameraGain"
+      name="Camera gain"
+      min="0"
+      max="100"
+      tooltip="Controls camera gain, similar to brightness"
+      :slider-cols="largeBox"
+      @input="handlePipelineData('cameraRedGain')"
+      @rollback="e => rollback('cameraRedGain', e)"
+    />
     <CVslider
       v-if="cameraRedGain !== -1"
       v-model="cameraRedGain"
-      name="Red AWB Gain"
+      name="Red Balance"
```

Comment:
Naming thoughts... "Red Gain"? Gain implies an orthogonal operation (like, just impacting red by itself) while Balance implies modifying multiple things at once to skew the image coloring.... idk what this is actually doing under the hood (or, if it's consistent across cameras?) and would be happy with either though.

Reply:
Yeah internally it's adjusting the target of the awb weighting to be more tinted red or more tinted blue -- down the line we may wanna add an auto awb mode as well